bower.json

```diff
@@ -31,7 +31,7 @@
     "webapp-module-logger": "https://github.com/wireapp/webapp-module-logger.git#1.1.0",
     "webapp-module-modal": "https://github.com/wireapp/webapp-module-modal.git#1.0.0",
     "webapp-module-namespace": "https://github.com/wireapp/webapp-module-namespace.git#1.0.0",
-    "webrtc-adapter": "https://github.com/webrtc/adapter.git#2.0.8",
+    "webrtc-adapter": "https://github.com/webrtc/adapter.git#2.1.0",
```

Comment:
I tried. For now we cannot use any of the build v3 builds. Thse contain major changes to the `RTCPeerConnection.getStats()` handling. Chrome will soon me more standards compliant and they have changed the shimming. The code is not yet able to handle these changes and all our flow checks thuse fail.